code_schemes/imaqal_special_show.json

```diff
@@ -3,6 +3,78 @@
   "Name": "Imaqal Special Show",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "StringValue": "ali",
+      "DisplayText": "ali",
+      "VisibleInCoda": true,
+      "NumericValue": 0,
+      "CodeID": "code-6c88bf5a",
+      "CodeType": "Normal"
+    },
+    {
+      "NumericValue": 1,
+      "CodeID": "code-9c9847a6",
+      "CodeType": "Normal",
+      "StringValue": "ismail",
+      "DisplayText": "ismail",
+      "VisibleInCoda": true
+    },
+    {
+      "NumericValue": 2,
+      "CodeID": "code-4cd72bce",
+      "CodeType": "Normal",
+      "StringValue": "mumina",
+      "DisplayText": "mumina",
+      "VisibleInCoda": true
+    },
+    {
+      "NumericValue": 3,
+      "CodeID": "code-10ad4f77",
+      "CodeType": "Normal",
+      "StringValue": "positive_remarks_about_drama",
```

Comment:
Is this the best order? Without having the context of the show + question, I would have thought it would make sense to list the names, all_names, then the positive remarks.

Reply:
i agree to. updated but kept numeric values as this is already in coda

Reply:
thanks